sort_sorttype.go

```diff
@@ -19,10 +19,9 @@ type chunk struct {
 	less CompareLessFunc
 }
 
-func newChunk(size int, lessFunc CompareLessFunc) *chunk {
```

Comment:
If you happen to know the chunk size in advance this offers a huge performance gain by not resizing the array continuously. You don't need to always fill the array and the chunk will still work.